coalib/settings/ConfigurationGathering.py

```diff
@@ -283,6 +288,13 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    for section_name in exclude_sections:
+        if section_name in sections:
+            del sections[section_name]
+        else:
+            logging.warning('The {} section doesn\'t exist, ignoring it.'
```

Comment:
please use double quotes for the string:
`"The {} ..."`
then you don't have to escape `'`